EFCore.SqlServer.VectorSearch/EFCore.SqlServer.VectorSearch.csproj

```
@@ -2,15 +2,15 @@
 
   <PropertyGroup>
     <Version>0.2.0</Version>
-    <TargetFramework>net8.0</TargetFramework>
+    <TargetFramework>net9.0</TargetFramework>
```

EF Core 9.0 targets net8.0, so the vector search plugin should as well.

```suggestion
    <TargetFramework>net8.0</TargetFramework>
```